bin/test/i18n/test_redact_restore_utils.rb

```diff
@@ -33,18 +37,28 @@ def test_redaction_of_json_file
     expected_source_data = 'expected_source_data'
     expected_dest_dir_path = 'expected_dest_dir'
     expected_dest_path = "#{expected_dest_dir_path}/dest.json"
-    expected_dest_file = mock
     expected_plugins = %w[testPlugin]
     expected_format = 'txt'
     expected_redacted_data = 'expected_redacted_data'
-    expected_redacted_json = '"expected_redacted_data"'
 
-    JSON.stubs(:load_file).with(expected_source_path).returns(expected_source_data)
+    I18nScriptUtils.stubs(:parse_file).with(expected_source_path).returns(expected_source_data)
     RedactRestoreUtils.stubs(:redact_data).with(expected_source_data, expected_plugins, expected_format).returns(expected_redacted_data)
-    File.stubs(:open).with(expected_dest_path, 'w+').yields(expected_dest_file)
 
-    FileUtils.expects(:mkdir_p).with(expected_dest_dir_path).once
-    expected_dest_file.expects(:write).with(expected_redacted_json).once
+    I18nScriptUtils.stubs(:write_json_file).with(expected_dest_path, expected_redacted_data)
+
+    RedactRestoreUtils.redact(expected_source_path, expected_dest_path, expected_plugins, expected_format)
+  end
+
+  def test_redaction_of_markdown_file
+    expected_source_path = MARKDOWN_FIXTURE_PATH
+    expected_dest_path = "expected_dest_dir/dest.json"
```

Comment:
why are we expecting a markdown file input to produce a json output?

Reply:
Good catch!